src/strings/en-us.json

```diff
@@ -1041,6 +1041,7 @@
     "OptionAlbumArtist": "Album Artist",
     "OptionAllUsers": "All users",
     "OptionAllowAudioPlaybackTranscoding": "Allow audio playback that requires transcoding",
+    "OptionForceRemoteSourceTranscoding": "Force remote media sources transcoding"
```

Comment:
You are missing a comma at the end of this line.

Reply:
fixed